types/howler/index.d.ts

```diff
@@ -90,6 +99,7 @@ interface Howl {
     on(event: 'seek', callback: (soundId: number) => void, id?: number): this;
     on(event: 'fade', callback: (soundId: number) => void, id?: number): this;
     on(event: string, callback: Function, id?: number): this;
+    on(event: 'onunlock', callback: (soundId: number) => void, id?: number): this;
```

Comment:
I don't think this is correct, the string should just be 'unlock'. Same with line 117. If you use onunlock as the event name, it will throw errors. It works correctly using 'unlock' which is the correct event name.

Reply:
Sorry, fixed!